src/Core/Managed/Shared/Extensibility/Implementation/OperationTelemetry.cs

```diff
@@ -81,6 +81,14 @@ public DateTimeOffset StartTime
         internal long BeginTimeInTicks { get; set; }
 
         /// <summary>
+        /// Sets random operation Id.
+        /// </summary>
+        public void GenerateOperationId()
```

Comment:
There was a public extension method in OperationTelemetryExtensions and this is a small refactoring that substitutes it with class method. DependencyTelemetry and RequestTelemetry initialized Id with `Convert.ToBase64String(BitConverter.GetBytes(WeakConcurrentRandom.Instance.Next()))` and now call `GetOperationId`

Ican make it internal since there is still extension method (in case someone called OperationTelemetryExtensions.GenerateOperationId(operation)).

Throwing exception would change behavior the extension method has now.